sources/Debate-frontend/src/app/loginpage/loginpage.component.html

```diff
@@ -1,14 +1,16 @@
 <div class="loginContent">
-    <form class="login-form" (ngSubmit)="onSubmit()">
-        <mat-form-field class="example-full-width">
-            <mat-label>Login</mat-label>
-            <input matInput placeholder="Podaj swój login" [(ngModel)]="model.login" name="login">
-        </mat-form-field>
-        <mat-form-field class="example-full-width">
-            <mat-label>Hasło</mat-label>
-            <input matInput type="password" placeholder="Wprowadź swoje hasło" [(ngModel)]="model.password" name="password">
-        </mat-form-field>
-        <button mat-raised-button color="primary" type="submit" style="margin-bottom: 7em; width: 30%; margin-left: 5px;" >Zaloguj</button>
-        <button mat-button class="go-register-button" routerLink="/register">Nie masz jeszcze konta? Zarejestruj się</button>
-    </form>
+    <div class="logCard">
+        <form class="login-form" #loginform="ngForm" (ngSubmit)="onSubmit()">
+            <mat-form-field class="example-full-width">
```

Comment:
Classess should be more meaningful like form-field, form-control or two separate "w100 text-center" rather than example-full-width